src/algebra/order/ring.lean

```diff
@@ -729,14 +741,6 @@ le_of_not_gt (λ ha, absurd h (mul_neg_of_pos_of_neg ha hb).not_le)
 lemma nonpos_of_mul_nonneg_right (h : 0 ≤ a * b) (ha : a < 0) : b ≤ 0 :=
 le_of_not_gt (λ hb, absurd h (mul_neg_of_neg_of_pos ha hb).not_le)
```

Comment:
Can you add a todo comment above these two lemmas that says the left/right is backwards compared to the `neg_of_mul_pos` versions?